src/geometry/euclidean/triangle.lean

```diff
@@ -316,6 +316,8 @@ begin
   { exact (vsub_sub_vsub_cancel_right p1 p3 p2).symm }
 end
 
+alias dist_sq_eq_dist_sq_add_dist_sq_sub_two_mul_dist_mul_dist_mul_cos_angle ← law_cos
```

Comment:
@jcommelin Does this need to be `euclidean_geometry.law_cos`?

Reply:
I think it's fine. But we can fix it later if it becomes a problem.